custom_components/tahoma/climate_devices/atlantic_heat_recovery_ventilation.py

```diff
@@ -232,26 +219,35 @@ def fan_modes(self) -> Optional[List[str]]:
     async def async_set_fan_mode(self, fan_mode: str) -> None:
         """Set new target fan mode."""
         if fan_mode == FAN_BYPASS:
-            await self.async_execute_command(
-                COMMAND_SET_AIR_DEMAND_MODE, "auto"
-            )
-            VENTILATION_MODE_STATE["cooling"] = "on"
-            await self.async_execute_command(
-                COMMAND_SET_VENTILATION_MODE, VENTILATION_MODE_STATE
-            )
+            await self.async_execute_command(COMMAND_SET_AIR_DEMAND_MODE, "auto")
         else:
-            if VENTILATION_MODE_STATE["cooling"] == "on":
-                VENTILATION_MODE_STATE["cooling"] = "off"
-                await self.async_execute_command(
-                    COMMAND_SET_VENTILATION_MODE, VENTILATION_MODE_STATE
-                )
             await self.async_execute_command(
-                COMMAND_SET_AIR_DEMAND_MODE, FAN_MODE_TO_TAHOMA[fan_mode]
+                COMMAND_SET_AIR_DEMAND_MODE, FAN_MODES_TO_TAHOMA[fan_mode]
```

Comment:
If cooling is "on", you have to set it to "off" before change the fan_mode

Reply:
Same comment as above, if cooling == on, the fan_mode is set to BYPASS. This part of the code won't be triggered in that case.